frontend/components/FilterContent.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="wrapper">
+    <div class="drpDwnBtnSect">
+      <div class="headfields">
+        <CourseSelector class="headfield" />
+        <CitySelector class="headfield" />
+        <PaidSelector class="headfield paidSelector" />
+      </div>
+      <hr />
+      <p class="tagTitleText noselect">
+        # Wybierz najważniejszą technologię z tagów
+      </p>
+      <div class="tagsContainer noselect">
+        <div class="tag"><span>React</span></div>
+        <div class="tag"><span>JS</span></div>
+        <div class="tag"><span>Android</span></div>
+        <div class="tag"><span>PHP</span></div>
+        <div class="tag"><span>HTML/CSS</span></div>
+        <div class="tag"><span>Data Analytics</span></div>
+        <div class="tag"><span>C++</span></div>
+        <div class="tag"><span>Python</span></div>
+        <div class="tag"><span>Game Dev</span></div>
+        <div class="tag"><span>Ruby</span></div>
+        <div class="tag"><span>Java</span></div>
+        <div class="tag"><span>.Net</span></div>
+        <div class="tag"><span>Scala</span></div>
+        <div class="tag"><span>Mobile</span></div>
+        <div class="tag"><span>Testing</span></div>
+        <div class="tag"><span>DevOps</span></div>
+        <div class="tag"><span>Security</span></div>
+        <div class="tag"><span>Scala</span></div>
```

Comment:
I would add this data to a variable or JSON file, I think it will grow in the future.

Reply:
It will be in JSON file, we just put some static tags for testing.

Reply:
Okey, for testing you can use JSON test file or array with data.

Reply:
Okay, You're right. I will change it to some test JSON file.

Reply:
Okay, I commited the changes.